docs/rules/no-extraneous-dependencies.md

```diff
@@ -89,7 +89,7 @@ var foo = require('./foo');
 
 import test from 'ava';
 import find from 'lodash.find';
-import find from 'lodash.isarray';
+import isArray from 'lodash.isarray';
```

Comment:
Instead tho, maybe we should not even bother importing this, since `Array.isArray` is in the language :-)

can we replace this with `omit` or something, since that's actually useful?